cpu_usage/package.ini

```diff
@@ -0,0 +1,11 @@
+[cpu_usage]
+uuid = hc1e3627-29e0-46fb-bf52-82bcfc8420f6

```

Comment:
This is not a valid UUID.

Reply:
Thank you Shawn. The uuid has been updated.